apps/condo/domains/news/components/NewsForm/BaseNewsForm.tsx

```diff
@@ -41,23 +51,48 @@ import { searchOrganizationProperty } from '@condo/domains/ticket/utils/clientSc
 import { SectionNameInput } from '@condo/domains/user/components/SectionNameInput'
 import { UnitNameInput, UnitNameInputOption } from '@condo/domains/user/components/UnitNameInput'
 
-export const NewsPreview = styled(MemoizedNewsPreview)`
-    color: turquoise;   
-`
-const CounterWrapper = styled.div`
-    position: absolute;
-    right: 0;
-    margin: 12px;
-    bottom: 0;
-    padding: 2px 10px;
-    background-color: black;
-    border-radius: 100px;
-`
+type FormWithActionChildrenProps = ComponentProps<ComponentProps<typeof FormWithAction>['children']>
+
+type ActionBarProps = Pick<FormWithActionChildrenProps, 'handleSave' | 'isLoading' | 'form'>
+
+type ActionNameProps = 'create' | 'update'
+
+type NewsItemClientUtilsType = IGenerateHooksResult<INewsItem, INewsItemCreateInput, INewsItemUpdateInput, IQueryAllNewsItemsArgs>
+
+export type SendPeriodType = 'now' | 'later'
+
+export type BaseNewsFormProps = {
+    organizationId: string
+    ActionBar: React.FC<ActionBarProps>
+    action: (values: INewsItemCreateInput | INewsItemUpdateInput) => ReturnType<ReturnType<NewsItemClientUtilsType['useCreate' | 'useUpdate']>>
+    initialValues?: INewsItem
+    & {
+        newsItemScopes: INewsItemScope[],
+        hasAllProperties: boolean,
+        sendPeriod: SendPeriodType,
+    },
+    templates: { [key: string]: Pick<INewsItemTemplate, 'title' | 'body'> }
+    afterAction?: () => void,
+    newsItem?: INewsItem,
+    OnCompletedMsg: (INewsItem) => ArgsProps | null,
+    allNews: INewsItem[],
+    actionName: ActionNameProps,
+}
+
+const COUNTER_COL_STYLE: React.CSSProperties = { position: 'absolute', right: 0, bottom: 0, margin: '12px', padding: '2px 10px', backgroundColor: 'black', borderRadius: '100px' }
```

Comment:
Does absolute positioning work properly at a narrow screen size?

Reply:
Yes, I need this style to style inputs counter. Attached below screenshot at narrow screen size

<img width="1121" alt="Screenshot 2023-05-31 at 10 33 54" src="https://github.com/open-condo-software/condo/assets/93817911/80718c61-231c-4ee1-83f8-a24547120721">